mod/01_rating/set/type/metric_value.rb

```diff
@@ -247,59 +245,65 @@ def legend args
-                                        :i,
-                                        '',
-                                        class: 'fa fa-caret-right
-                                                fa-lg margin-left-10
-                                                btn btn-default btn-sm',
-                                        data: { toggle: 'collapse-next',
-                                                parent: '.value',
-                                                collapse: '.metric-value-details'
-                                              }
-                                      )
-    credit = wrap_with :div, class: 'credit' do
+    value_details = _render_value_details(args)
+    year = content_tag(:span, card.cardname.right)
+    year << content_tag(:div, '', class: 'timeline-dot')
+    year = content_tag(:div, year.html_safe, class: 'td year')
+    value =  _render_modal_details(args)
+    value << content_tag(:span, legend(args), class: 'metric-unit')
+    value << _render_value_details_toggle
+    value << value_details.html_safe
+
+    # stitch together
+    wrap_with :div, class: 'timeline-row' do
       [
-        nest(card, view: :core, structure: 'creator credit'),
-        _optional_render(:source_link, args, :hide)
+        year,
+        content_tag(:div, value.html_safe, class: 'td value')
       ]
     end
-    # value_details =  content_tag(:div, credit.html_safe, class: )
-    # year parent container
-    year = content_tag(:span, card.cardname.right)
-    year << dot
-    year = content_tag(:div, year.html_safe,  class: 'td year')
+  end
 
-    # comments
-    comments = (disc_card = card.fetch trait: :discussion) &&
-               subformat(disc_card).render_core.html_safe
-    # source
-    sources = card.fetch trait: :source
-    sources = subformat(sources).render_core(item: :cited).html_safe
-    # sources = subformat(sources).render_core(item: :content, structure: 'source item').html_safe
-    value_details = wrap_with :div, class: 'metric-value-details collapse' do
+  view :value_details do |args|
+    comments = _render_comments
+    sources = _render_sources
+    wrap_with :div, class: 'metric-value-details collapse' do
       [
-        # line,
-        credit.html_safe,
+        _optional_render(:credit_name, args, :show),
         content_tag(:div, comments, class: 'comments-div'),
         content_tag(:div, sources, class: 'cited-sources')
       ]
     end
+  end
 
-    # value parent container
-    value =  _render_modal_details(args)
-    value << content_tag(:span, legend(args), class: 'metric-unit')
-    value << value_details_toggle.html_safe
-    value << value_details.html_safe
-    # line = content_tag(:div, '', class: 'timeline-line') if args[:connect]
+  view :value_details_toggle do
+    content_tag(:i, '', class: 'fa fa-caret-right
+                                fa-lg margin-left-10
+                                btn btn-default btn-sm',
```

Comment:
Use `\` for breaking strings otherwise you get linebreaks in your string
Like this:
``` 
  'sdfasdf' \
  'dsfasd' 
```